enterprise/api/license.go

```diff
@@ -54,4 +54,6 @@ type LicenseService interface {
 	StoreLicense(tokenString string) error
 	// LoadLicense will load license from file and validate it.
 	LoadLicense() (*License, error)
```

Comment:
Instead of having a new method, shall this be part of StoreLicense()?

Reply:
The logic is updated now, we won't store invalid license to avoid user upload by mistake.

Reply:
But I suddenly found if so we cannot switch to the free plan manually...Need to find a way to workaround

Reply:
delete the license?

Reply:
LGTM, I think it's better than adding a specific check if the license is an empty string.

Reply:
The check should still be part of StoreLicense?

Reply:
Updated, we don't need the new public method anymore